requirements.txt

```diff
@@ -1,5 +1,4 @@
 aiohttp>=3.8.1,<4
-cryptography>=42.0.4,<43
```

Comment:
That's on purpose ?

Reply:
Yes.
It is already taken from opal-common @ git+https://github.com/permitio/opal.git@master#subdirectory=packages/opal-common